app/views/extra_registrations/_setting.html.slim

```diff
@@ -1,5 +1,11 @@
 = f.govuk_fieldset legend: { text: t(:heading, scope: [:extra_registration, current_step]) } do
   p.govuk_body
     = t(:body, scope: [:extra_registration, current_step])
+  = f.govuk_radio_buttons_fieldset(:setting_type, legend: { text: t(:setting_type, scope: 'helpers.labels.setting_type'), size: 's'})
```

Comment:
https://govuk-form-builder.netlify.app/form-elements/radios/#populating-values-labels-and-hints-with-procs

FYI If we wanted to move things into locales it is possible

Reply:
Good idea for later on.

Reply:
Actually, I did look at that option, but it didn't work with the "other" with option to enter data.  If we can get rid of 'other' I think it would work.

Reply:
Or maybe the 'other' option needs to be separated from the collection....